### ISSUE TITLE
New Issue: Can't import and register universe environments did something break?

### ISSUE THREAD

**Issue by tneksenoj** [openai/gym#853]
In the following example code, the i**mport universe** statement is now failing.  It worked for me at the start of January but now its not working anymore. Any ideas what might be wrong? Is the scoreboard being removed from universe and not all the code updated yet? Any help is appreciated!

######################################################
## HERE IS THE PYTHON CODE:

# Adapted from the official example
# See http://terokarvinen.com/2016/install-openai-universe-on-ubuntu-16-04
import sys
assert sys.version_info[0]==3 # python3 required

import gym
import universe  # register the universe environments

env = gym.make('flashgames.DuskDrive-v0')
env.configure(remotes=1)  # automatically creates a local docker container
observation_n = env.reset()

while True:
 action_n = [[('KeyEvent', 'ArrowUp', True)] for ob in observation_n]  # your agent here
 observation_n, reward_n, done_n, info = env.step(action_n)
 env.render()

######################################################
## HERE IS THE ERROR:

Traceback (most recent call last):
  File "./tbot.py", line 7, in <module>
    import universe  # register the universe environments
  File "/home/domain/username/openai/universe/universe/__init__.py", line 20, in <module>
    import universe.scoreboard
  File "/home/domain/username/openai/universe/universe/scoreboard/__init__.py", line 1, in <module>
    from gym.benchmarks import scoring
ImportError: No module named 'gym.benchmarks'

########################################################
## HERE IS MY SETUP

Linux MACHINE 4.13.0-32-generic #35~16.04.1-Ubuntu SMP Thu Jan 25 10:13:43 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
Python 3.5.2
Name: universe
Version: 0.21.5
Summary: Universe: a software platform for measuring and training an AI's general intelligence across the world's supply of games, websites and other applications.
Home-page: https://github.com/openai/universe
Author: OpenAI
Author-email: universe@openai.com
License: UNKNOWN
Location: /home/domain/username/universe
Requires: autobahn, docker-py, docker-pycreds, fastzbarlight, go-vncdriver, gym, Pillow, PyYAML, six, twisted, ujson
---
Name: gym
Version: 0.9.6
Summary: The OpenAI Gym: A toolkit for developing and comparing your reinforcement learning agents.
Home-page: https://github.com/openai/gym
Author: OpenAI
Author-email: gym@openai.com
License: UNKNOWN
Location: /home/domain/username/openai/gym
Requires: numpy, requests, six, pyglet
---
Name: numpy
Version: 1.14.0
Summary: NumPy: array processing for numbers, strings, records, and objects.
Home-page: http://www.numpy.org
Author: NumPy Developers
Author-email: numpy-discussion@python.org
License: BSD
Location: /home/domain/username/openai/env/lib/python3.5/site-packages
Requires: 
---
Name: go-vncdriver
Version: 0.4.19
Summary: UNKNOWN
Home-page: UNKNOWN
Author: UNKNOWN
Author-email: UNKNOWN
License: UNKNOWN
Location: /home/domain/username/openai/go-vncdriver
Requires: numpy
---
Name: Pillow
Version: 5.0.0
Summary: Python Imaging Library (Fork)
Home-page: https://python-pillow.org
Author: (omitted)
Author-email: (omitted)
License: Standard PIL License
Location: /home/domain/username/openai/env/lib/python3.5/site-packages
Requires:

**Comment by tneksenoj** [comment]
I found the answer to my own question.  Look at Accessant's answer to issue 849 ( https://github.com/openai/gym/issues/849 ) and I quote: " ...Also note: the newest version of gym got rid of the benchmarks functions, which universe uses. Wait for universe update or install 0.9.5."

**Comment by GokulNC** [comment]
In short, uninstall present gym by`pip uninstall gym`, download and extract [gym 0.9.5 from here](https://pypi.python.org/pypi/gym/0.9.5), and install as per the instructions in [the gym repo](https://github.com/openai/gym/)

Or if you wan to use the latest version of Gym, wait for the corresponding universe update to support it as mentioned above.
